src/sdk/services/reactors.ts

```diff
@@ -1,8 +1,10 @@
+import { RequestOptions } from '.';
```

Comment:
```suggestion
import { RequestOptions } from './shared';
```
Importing from same index where types / consts are also exported may lead to circular dependencies in some edge cases..